react/features/app/actions.js

```diff
@@ -49,6 +47,15 @@ export function appNavigate(uri) {
                     .domain;
         }
 
+        if (room) {
```

Comment:
How does a URL without a scheme end up there? Because the user typed it like that?